stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/SectionElementUI.kt

```diff
@@ -1,4 +1,6 @@
-package com.stripe.android.ui.core.elements
+@file:RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
Why is this needed?

Reply:
With this we avoid files showing up in our `api` files : )